pylxd/image.py

```diff
@@ -84,3 +84,33 @@ def export(self):
         """Export the image."""
         response = self.api.export.get()
         return response.content
+
+    def add_alias(self, name, description):
+        """Add an alias to the image."""
+        self.client.api.images.aliases.post(json={
+            'description': description,
+            'target': self.fingerprint,
+            'name': name
+        })
+
+        # Update current aliases list
+        self.aliases.append({
+            'description': description,
+            'target': self.fingerprint,
+            'name': name
+        })
+
+        return self
```

Comment:
Per the other review, why would we need to return self here? It's an instance method, so I'm unsure what the reasoning was here.